warehouse/query-core/src/main/java/datawave/query/jexl/functions/GeoWaveFunctionsDescriptor.java

```diff
@@ -38,7 +44,7 @@
 
 /**
  * This is the descriptor class for performing geowave functions. It supports basic spatial relationships, and decomposes the bounding box of the relationship
- * geometry into a set of geowave ranges. It currently caps this range decomposition to 800 ranges and in practice should be considerably less.
+ * geometry into a set of geowave ranges. It currently caps this range decomposition to 8 ranges per tier.
```

Comment:
I thought we were doing 32 if a point?

Reply:
It is.  There is a default for geometry types, and a separate default for point types.  Take a look at this:
https://github.com/NationalSecurityAgency/datawave/pull/447/files/93fdf4c58f4a320fd598cffc96571198f55d7346#diff-bbccefcacb3fc24482b6c275fb1aa5d2R146